### PR TITLE
Create a python openstack webserver example

### PR DESCRIPTION
**Pull request by kbaikov**
Shows how to create an openstack instance
starting with image upload, addition of
security group rules, creating a keypari for
ssh access and a custom user data
script to run on the instance

### CODE REVIEW

File: openstack-py-webserver/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+4. View the host name and IP address of the instance via `stack output`:
+
+    ```bash
+    $ pulumi stack output
+    Current stack outputs (1):
+        OUTPUT       VALUE
+        instance_ip  192.168.0.243
+    ```
+
+5.  Verify that the EC2 instance exists, by either using the AWS Console or running `openstack server list`.
```
Comment (jaxxstorm) [inline openstack-py-webserver/README.md:59]: I think you mean OpenStack instance here?

**Review by jaxxstorm — APPROVED** [review]

File: openstack-py-webserver/README.md (hunk 1 above)
Comment (kbaikov) [reply openstack-py-webserver/README.md]: True, Thank you.

**Comment by kbaikov** [comment]
@jaxxstorm Fixed the requested text.
Do you need to run the acceptance tests again?
Do i need to react to some failed aws test?

**Comment by stack72** [comment]
Merging this as @jaxxstorm has approved

**Pull request merged**